ui/src/components/index.tsx

```diff
@@ -16,7 +16,6 @@ export { default as BookmarkedPublications } from './Publication/BookmarkedPubli
 export { default as BookmarkedTopics } from './Topic/BookmarkedTopics';
 export { default as Button } from './Button';
 export { default as ContentSection } from './GenericContent/ContentSection';
-export { default as Delay } from './Delay';
```

Comment:
Removed this component with PO's approval because it didn't especially "feel" worse without it and it was annoying to write tests around.